docs/Tutorials/Developer-Quickstart.md

```diff
@@ -29,4 +32,5 @@ npx quorum-dev-quickstart
 Follow the prompts displayed.
 
 When installation is complete, refer to `README.md` in the installation directory for more information
-on your test network.
+on your test network. Optionally refer to the walkthrough above
```

Comment:
"Optionally, refer to the [guide/walk-through] above."

Remove the extra line below this line to fix the markdownlint error.

Reply:
done